src/pages/index.js

```diff
@@ -1,38 +1,75 @@
 import {
+  apiOptions,
   validationSettings,
+  formList,
+  profileName,
+  profileDescription,
+  profileAvatar,
   profileEditButton,
   addNewCardButton,
-  initialCards,
+  avatarEditButton,
 } from "../utils/constants.js";
 import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
 import UserInfo from "../components/UserInfo.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import PopUpWithImage from "../components/PopupWithImage.js";
+import PopupWithConfirmation from "../components/PopupWithConfirmation.js";
 import Section from "../components/Section.js";
+import Api from "../components/API.js";
 import "./index.css";
 
 /* -------------------------------------------------------------------------- */
 /*                                 Form add and Validation                    */
 /* -------------------------------------------------------------------------- */
 
-const formValidators = {}
+const formValidators = {};
 
 // enable validation
 const enableValidation = (validationSettings) => {
-  const formList = Array.from(document.querySelectorAll(validationSettings.formSelector))
   formList.forEach((formElement) => {
-    const validator = new FormValidator(validationSettings, formElement)
-    const formName = formElement.id
+    const validator = new FormValidator(validationSettings, formElement);
+    const formName = formElement.id;
     formValidators[formName] = validator;
-   validator.enableValidation();
+    validator.enableValidation();
   });
 };
-
 enableValidation(validationSettings);
 
 /* -------------------------------------------------------------------------- */
-/*                               Popup                                        */
+/*                              Create API Client                             */
+/* -------------------------------------------------------------------------- */
+
+const api = new Api(apiOptions);
+
+/* -------------------------------------------------------------------------- */
+/*                                   Section                                  */
+/* -------------------------------------------------------------------------- */
+
+const cardSection = [];
```

Comment:
Needs correction: use `let` here instead of `const`, since you reassign the value on line 54